blocks/contrast-checker/test/index.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * External dependencies
+ */
+import { shallow } from 'enzyme';
+import tinycolor from 'tinycolor2';
+
+/**
+ * Internal dependencies
+ */
+import ContrastChecker from '../';
+
+describe( 'ContrastChecker', () => {
+	const backgroundColor = '#ffffff';
+	const textColor = '#000000';
+	const isLargeText = true;
+	const fallbackBackgroundColor = '#fff';
+	const fallbackTextColor = '#000';
+	const sameShade = '#666';
+
+	const wrapper = shallow(
+		<ContrastChecker
+			backgroundColor={ backgroundColor }
+			textColor={ textColor }
+			isLargeText={ isLargeText }
+			fallbackBackgroundColor={ fallbackBackgroundColor }
+			fallbackTextColor={ fallbackTextColor } />
+	);
+
+	test( 'should render null when no colors are provided', () => {
+		expect( shallow( <ContrastChecker /> ).html() ).toBeNull();
+	} );
+
+	test( 'should render null when the colors meet AA WCAG guidelines.', () => {
```

Comment:
This test case is not covering anything specific to `ContrastChecker` ? Should we assume TinyColor is tested separately?

Reply:
I think we assume that TinyColor is tested separately. If the colors are `readable` and expect that to be `true`, the component does not render anything. The component honestly confused me a lot, because I didn't fully realize what it did, but I think the intent of the component is to not do anything when the accessibility is fine for the colors. So this test basically covers the most common use case.